src/Routes/ParticipantRoutes.tsx

```diff
@@ -63,6 +63,78 @@ export default [
     exact
     key="nhspreregistration"
   />,
+  <Route
+    path="/nhsenwl"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=poster&utm_medium=print&utm_campaign=nhsenwl"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhsenwl"
+  />,
+  <Route
+    path="/nhseswl"
+    render={() => {
+      ReactGA.pageview(
+        "/participants/introduction?utm_source=poster&utm_medium=print&utm_campaign=nhseswl"
+      );
+      return <NhsPreRegistration />;
+    }}
+    strict
+    exact
+    key="nhseswl"
+  />,
```

Comment:
2nd route is a duplicate of the 1st

Reply:
1st route is nhsenwl, 2nd is nhseswl, small change but they wanted all 6